scripts/src/main/resources/scripts/command/az

```diff
@@ -1,106 +1,77 @@
+  setLocation
+  if [ "${1}" != "silent" ] || [ ! -f "${AZ_FILE}" ] 
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
+    if [ -d "${PYTHON_HOME}" ] 
     then
-      doEcho "Azure CLI is already installed at $(command -v az)"
+      doDevonCommand python pip install azure-cli
+      setConfig
+    else
+      doDevonCommand python setup silent
+      doDevonCommand python pip install azure-cli
+      setConfig
     fi
-  else
-    doRequireNotBatch
-    if doIsWindows
-    then
-      # Get leatest release
-      if [ -z "${AZ_VERSION}" ]
-      then
-        doEcho "Getting latest release..."
-        AZ_VERSION=$(curl -s https://api.github.com/repos/Azure/azure-cli/releases/latest | awk -F ":" '/tag_name/ {print $2}'| awk -F "\"" '{print $2}' | awk -F "-" '{print $3}')
-      fi
+  fi
+  if [ "${1}" != "silent" ] && ! doIsQuiet
+  then
+    doRunCommand "az -v" "verify installation of azure cli"
+  fi
+}
 
-      doInstall "-" "${AZ_HOME}" "az" "${AZ_VERSION}"
-      if [ "${?}" = 0 ]
-      then
-        doPackageInstall "${AZ_HOME}"
-      fi
-    fi
+function setConfig() {
+  az_config_dir="${DEVON_IDE_HOME}/conf/.azure"
+  az_config_export="export AZURE_CONFIG_DIR=${az_config_dir}"
+  if ! grep -q "${az_config_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      doRunCommand "${az_config_export}"
+      echo -e "\n${az_config_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+      doEcho "Location of Azure's configuration file is set to ${az_config_dir}"
   fi
 }
 
-function doPackageInstall() {
-  local path_to_package="${1}"
+function setLocation() {
   if doIsWindows
   then 
-    local winPath
-    winPath=$(cygpath -w "${path_to_package}")
-    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${winPath}\az-${AZ_VERSION}-windows.msi /QB-!'" &> /dev/null
-    if [ "${?}" = 0 ]
-    then
-      setConfigDir
-      doSuccess "Azure CLI ist installed now. Great! \nTo get started you may need to restart your current shell."
-    else
-      doError "The setup was canceled or failed. Rerun the setup to install Azure CLI."
-    fi
+    AZ_FILE="${DEVON_IDE_HOME}/software/python/Scripts/az"
   else
-    doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
-  fi
-  doRunCommand "rm -rf ${path_to_package}"
-}
-    
-# Set directory for azure cli configuration file
-function setConfigDir() {
-  azureconf_dir="${DEVON_IDE_HOME}/conf/.azure"
-  azurecli_export="export AZURE_CONFIG_DIR=${azureconf_dir}"
-  if ! grep -q "${azurecli_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
-    then
-      doRunCommand "${azurecli_export}"
-      echo -e "\n${azurecli_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
-      doEcho "Location of Azure's configuration file is set to ${azureconf_dir}"
+    AZ_FILE="${DEVON_IDE_HOME}/software/python/bin/az"
```

Comment:
Same here. IMHO you can simply remove the OS switch and replace all with the approach I suggested above.